packages/client/src/app/pages/GameOver/GameOver.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Layout from "../Layout/Layout";
+import styles from './GameOver.module.scss'
+import Button, {ButtonVariant} from "../../components/core/Button/Button";
+import {useNavigate} from "react-router-dom";
+
+function GameOver() {
+    const navigate = useNavigate();
+    const navigateToStartHandler = () => {
+        navigate('/')
+    }
+
+    return (
+        <Layout withGradient>
+            <div className={styles['game-over-container-wrap']}>

```

Comment:
Тут конечно лучше у Игоря уточнить, но мне тяжело воспринимать такие длинные название (тем более в таких количествах. Здесь и далее напишу как бы я именовал:
```suggestion
            <div className={styles['content-wrap']}> 
```
 page|content|или любой другой вариант, желательно в одно слово